frontend/server/src/test/java/org/pytorch/serve/ModelServerTest.java

```diff
@@ -1133,7 +1134,11 @@ public void testModelWithInvalidCustomPythonDependency()
     @Test(
             alwaysRun = true,
             dependsOnMethods = {"testModelWithInvalidCustomPythonDependency"})
-    public void testLoadingMemoryError() throws InterruptedException {
+    public void testLoadingMemoryError() throws Exception {
```

Comment:
throws allows multiple exception types to be reported. Is there a reason why you changed the type to "Exception" instead of "InterruptedException, SkipException"?

Reply:
Thanks for the tip, yeah fixed that just now